src/nbl/video/vulkan/profiles/NablaCore.json

```diff
@@ -1,2017 +1,2216 @@
-                    ],
-                    "subgroupSupportedOperations": [
-                        "VK_SUBGROUP_FEATURE_BASIC_BIT",
-                        "VK_SUBGROUP_FEATURE_VOTE_BIT",
-                        "VK_SUBGROUP_FEATURE_ARITHMETIC_BIT",
-                        "VK_SUBGROUP_FEATURE_BALLOT_BIT",
-                        "VK_SUBGROUP_FEATURE_SHUFFLE_BIT",
-                        "VK_SUBGROUP_FEATURE_SHUFFLE_RELATIVE_BIT",
-                        "VK_SUBGROUP_FEATURE_QUAD_BIT"
-                    ],
-                    "subgroupQuadOperationsInAllStages": true,
-                    "maxMultiviewViewCount": 6,
-                    "maxMultiviewInstanceIndex": 134217727,
-                    "maxPerSetDescriptors": 572,
-                    "maxMemoryAllocationSize": 1474838528
-                },
-                "VkPhysicalDeviceVulkan12Properties": {
-                    "shaderSignedZeroInfNanPreserveFloat16": true,
-                    "shaderSignedZeroInfNanPreserveFloat32": true,
-                    "maxUpdateAfterBindDescriptorsInAllPools": 1048576,
-                    "shaderSampledImageArrayNonUniformIndexingNative": true,
-                    "shaderStorageImageArrayNonUniformIndexingNative": true,
-                    "shaderInputAttachmentArrayNonUniformIndexingNative": true,
-                    "robustBufferAccessUpdateAfterBind": true,
-                    "maxDescriptorSetUpdateAfterBindInputAttachments": 256,
-                    "maxDescriptorSetUpdateAfterBindSampledImages": 500000,
-                    "maxDescriptorSetUpdateAfterBindSamplers": 500000,
-                    "maxDescriptorSetUpdateAfterBindStorageBuffers": 500000,
-                    "maxDescriptorSetUpdateAfterBindStorageBuffersDynamic": 8,
-                    "maxDescriptorSetUpdateAfterBindStorageImages": 500000,
-                    "maxDescriptorSetUpdateAfterBindUniformBuffers": 90,
-                    "maxDescriptorSetUpdateAfterBindUniformBuffersDynamic": 8,
-                    "maxPerStageDescriptorUpdateAfterBindInputAttachments": 64,
-                    "maxPerStageDescriptorUpdateAfterBindSampledImages": 500000,
-                    "maxPerStageDescriptorUpdateAfterBindSamplers": 500000,
-                    "maxPerStageDescriptorUpdateAfterBindStorageBuffers": 500000,
-                    "maxPerStageDescriptorUpdateAfterBindStorageImages": 500000,
-                    "maxPerStageDescriptorUpdateAfterBindUniformBuffers": 15,
-                    "maxPerStageUpdateAfterBindResources": 500000,
-                    "supportedDepthResolveModes": [
-                        "VK_RESOLVE_MODE_SAMPLE_ZERO_BIT",
-                        "VK_RESOLVE_MODE_MAX_BIT"
-                    ],
-                    "supportedStencilResolveModes": [
-                        "VK_RESOLVE_MODE_SAMPLE_ZERO_BIT"
-                    ],
-                    "independentResolveNone": true,
-                    "independentResolve": true,
```

Comment:
something is very wrong in some report you took as input. it probably has `VkPhysicalDeviceDepthStencilResolvePropertiesKHR` where it reports `independentResolveNone = true` but here for the Vulkan core feature it doesn't

Reply:
the shitty github ui tricked you. `independentResolveNone` and `independentResolve` are all set to `true` in all profiles